### ISSUE TITLE
Ashstorm Runtime

### ISSUE THREAD

**Issue by AlinaStarkova** [tgstation/tgstation#60034]
## Reproduction:
(only the pipenet issue in this report is still ongoing)

I have my fork updated to the latest (hard reset to tgs master like an hour ago) and every single time i load up 2, and sometimes 3 things happen

first, midloading i get this (will be fixed in #60366):
```
## WARNING: design /datum/design/buffer_upgrade with icon 'icons/obj/items_and_weapons.dmi' missing state 'janicart_upgrade'
## WARNING: design /datum/design/vacuum_upgrade with icon 'icons/obj/items_and_weapons.dmi' missing state 'janicart_upgrade'
```

second, this one happened only twice out of the 7 times ive reloaded my local host, so this one doesnt happen ALL the time but does sometimes:
`## WARNING: Component is missing a pipenet! Rebuilding... in code/modules/atmospherics/machinery/components/components_base.dm at line 205 src: the grey iron pressure tank (Air) usr: .`
^ this only spams me about 20 messages then stops

but the big thing is once the servers done loading fully, i get spammed with this NONSTOP until i close the server (fixed in #60043):
```
Initialized Chat subsystem within 0 seconds!
Initializations complete within 77.4 seconds!
Game start took 0.9s
[01:11:23] Runtime in weather.dm,195: Cannot read null.z
  proc name: can weather act (/datum/weather/proc/can_weather_act)
  src: ash storm (/datum/weather/ash_storm)
  call stack:
  ash storm (/datum/weather/ash_storm): can weather act(Nari Sashimi (/mob/living/carbon/human/dummy))
  Weather (/datum/controller/subsystem/weather): fire(0)
  Weather (/datum/controller/subsystem/weather): ignite(0)
  Master (/datum/controller/master): RunQueue()
  Master (/datum/controller/master): Loop()
  Master (/datum/controller/master): StartProcessing(0)
```
^ this happens on every load as well, again, im running the latest master branch from tg, nothing is different

**Comment by Greniza** [comment]
ah crap, I think the latter of those is mine. I'll get it fixed.

**Comment by timothymtorres** [comment]
This was fixed with #60043
